staging_service/app.py

```diff
@@ -462,7 +450,7 @@ async def upload_files_chunked(request: web.Request):
             f.write(chunk)
 
     if not os.path.exists(path.full_path):
-        error_msg = "We are sorry but upload was interrupted. Please try again.".format(
+        error_msg = "We are sorry but upload was interrupted. Please try again.".format(  # noqa F522
```

Comment:
just remove the "format" part as the `path` variable never gets interpolated

i.e.
```py
error_msg = "We are sorry but upload was interrupted. Please try again."
```